src/driver/op.rs

```diff
@@ -133,49 +153,82 @@ where
             Lifecycle::Completed(result, flags) => {
                 inner.ops.remove(me.index);
                 me.index = usize::MAX;
-
                 Poll::Ready(me.data.take().unwrap().complete(result, flags))
             }
+            Lifecycle::CompletionList(..) => {
```

Comment:
This can be safely ignore. By implmenting a SingleCQE class of operation, we're promising we'll never receive a CQE with more set, so this can never be constructed